src/components/CategoryNavTabs/CategoryNavTabs.jsx

```diff
@@ -0,0 +1,30 @@
+import "./categoryNavTabs.css";
+
+function CategoryNavTab() {
+  return (
+    <section class="categories_navTab_container">
+      <ul class="categories_navTabs_list">
+        <li>
+          <a
+            class="categories_navLink categories_navLink1"
+            href="/new-arrivals"
+          >
+            New Arrivals
+          </a>
+        </li>
+        <li>
+          <a class="categories_navLink categories_navLink2" href="/bestsellers">
+            Best Sellers
+          </a>
+        </li>
+        <li>
+          <a class="categories_navLink categories_navLink2" href="/trending">
+            Trending
+          </a>
+        </li>
+      </ul>
+    </section>
+  );
+}
+
+export default CategoryNavTab;
```

Comment:
You can add an extra line at the end of each file then it will not show this end tag

Reply:
Can you explain again , I didnt get your point , are yu talking about this red icon kind of thing

Reply:
Yess I will do it from next time